Makefile

```diff
@@ -84,6 +108,12 @@ docker-push: ## Push docker image with the manager.
 photon:
 	docker build -t github.com/vmware-tanzu/nsx-operator -f build/image/photon/Dockerfile .
 
+.PHONY: cleanall
+cleanall:
+	@rm -rf $(BINDIR)
+	@rm -rf $(GOLANGCI_LINT_BINDIR)
+	@rm -rf $(CURDIR)/.coverage
+
 ##@ Deployment
```

Comment:
could we move Line93 
go build -o bin/clean cmd_clean/main.go
to build target?
And rename cleanall to clean

Reply:
how about we build cmd/main.go and cmd_clean/main.go both in "make build" target and change this to "clean" target?